lib/__tests__/index.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+const assert = require('assert');
+const del = require('del');
+const fs = require('fs');
+const metalToolsSoy = require('../../index');
+const sinon = require('sinon');
+const vfs = require('vinyl-fs');
+
+describe('Metal Tools - Soy', function() {
+	beforeEach(function() {
+		const stream = {
+			pipe: function() {
+				return stream;
+			},
+			readable: true,
+			resume: sinon.stub(),
+		};
+		sinon.stub(vfs, 'src').returns(stream);
+		sinon.stub(vfs, 'dest');
+	});
+
+	afterEach(function() {
+		restoreStream();
+	});
+
+	it('should compile soy files from/to "src" folder by default', function() {
```

Comment:
I think this is a more of a stylistic choice that jest provides. I don't think they have any plans to deprecate `it`.